bindings/java/src/test/java/org/openucx/jucx/UcpListenerTest.java

```diff
@@ -111,26 +118,15 @@ public void testConnectionHandler() throws Exception {
 
         // Test connection handler persists
         for (int i = 0; i < 10; i++) {
-            conRequest.set(null);
+            AtomicBoolean rejected = new AtomicBoolean(false);
             UcpEndpoint tmpEp = clientWorker.newEndpoint(new UcpEndpointParams()
                 .setSocketAddress(serverListener.getAddress()).setPeerErrorHandlingMode()
-                .setErrorHandler((ep, status, errorMsg) ->
```

Comment:
maybe not remove this test?
because the connection could be rejected also if for some reason the connection handler is not called